src/components/Buildbaron/BuildBaronContent.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { useEffect, useState } from "react";
+import styled from "@emotion/styled";
+import { Subtitle } from "@leafygreen-ui/typography";
+import { useParams } from "react-router-dom";
+import { Banners } from "components/Banners";
+import {
+  useBannerDispatchContext,
+  useBannerStateContext,
+} from "context/banners";
+import { BuildBaron } from "gql/generated/types";
+import { withBannersContext } from "hoc/withBannersContext";
+import { BBTitle, TitleAndButtons } from "./BBComponents";
+import { BBFileTicket, CreatedTickets } from "./BBFIleTicket";
+import { BuildBaronTable } from "./BuildBaronTable";
+
+interface Props {
+  eventData: BuildBaron;
+  taskId: string;
+}
+
+const BuildBaronCore: React.FC<Props> = ({ eventData, taskId }) => {
+  const dispatchBanner = useBannerDispatchContext();
+  const bannersState = useBannerStateContext();
+  const { tab } = useParams<{ tab: string }>();
+  useEffect(() => {
+    dispatchBanner.clearAllBanners();
+  }, [tab]); // eslint-disable-line react-hooks/exhaustive-deps
+  const [createdTicketsCount, setCreatedTicketsCount] = useState<number>(0);
+  console.log(`BuildBaronContent 29: ${createdTicketsCount}`);
+  return (
+    <>
+      {eventData && (
+        <>
+          <Banners
+            banners={bannersState}
+            removeBanner={dispatchBanner.removeBanner}
+          />
+
+          <CreatedTickets
+            taskId={taskId}
+            dispatchBanner={dispatchBanner}
+            setCreatedTicketsCount={setCreatedTicketsCount}
+            createdTicketsCount={createdTicketsCount}
+          />
+
+          <TitleAndButtons>
+            <BBFileTicket
+              taskId={taskId}
+              dispatchBanner={dispatchBanner}
+              setCreatedTicketsCount={setCreatedTicketsCount}
+              createdTicketsCount={createdTicketsCount}
+            />
+            <BBTitle>Related tickets from Jira </BBTitle>
+          </TitleAndButtons>
+          <BuildBaronTable jiraIssues={eventData?.searchReturnInfo?.issues} />
+        </>
+      )}
+    </>
+  );
+};
+export const StyledSubtitle = styled(Subtitle)<titleProps>`
+  margin-bottom: ${(props) => (props.margin ? "15px" : "5px")};
+  margin-top: ${(props) => (props.margin ? "25px" : "20px")};
+  font-size: 16px;
```

Comment:
I think we've been avoiding custom font sizes for these predefined typography components.

Reply:
Okay, thanks. I'll look for predefined styles I can use. It may not look quite like the design though.

Reply:
![image](https://user-images.githubusercontent.com/13104717/93812921-4aa98c80-fc20-11ea-854a-56b36d931661.png)
This is what it looks like if I remove it. It's 18px without overriding it. I does not look like the design, but probably close enough.